src/universal/modules/userDashboard/components/UserActionList/UserActionListItem.js

```diff
@@ -60,7 +59,7 @@ class UserActionListItem extends Component {
           <Field
             name={actionId}
             component={OutcomeCardTextarea}
-            doFocus={!content}
+            doSubmitOnEnter
```

Comment:
The Action list felt funny without being able to submit on enter:

![image](https://cloud.githubusercontent.com/assets/103000/22368895/ecb10ad4-e456-11e6-8de6-60a3d66f70c4.png)